<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adcExt</class>
 <widget class="QWidget" name="adcExt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="adcExtbackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analog Inputs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readAdcButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1. Cabinet Temp.(degC)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2. Not Used</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3. Shunt_Nozzle2 (Amp)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>226</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4. Shunt_Nozzle1 (Amp)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>155</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5. System Voltage1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>155</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6. System Voltage2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>176</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7. DC+ Temp.(degC)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_an8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8. DC- Temp.(degC)</string>
    </property>
   </widget>
  </widget>
  <zorder>adcExtbackButton</zorder>
  <zorder>label</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>readAdcButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
